cron/format/json_raw_results.go

```diff
@@ -0,0 +1,308 @@
+// Copyright 2022 Security Scorecard Authors
```

Comment:
Any reason there is separate logic for this in `cron`? This should ideally be the same JSON as exposed to users in `checker` package.

Reply:
like the existing json-based results, this gives us a chance to update the raw result format for the scorecard, without breaking the cron format. For now this is simply going to be the same code, though

Reply:
> update the raw result format for scorecard, without breaking the cron format. 

Yeah, but the raw JSON schema will be publicly available and the cron job should honor that.

Ok for now, but ideally this shouldn't diverge.